pronto/api/signature.py

```diff
@@ -297,6 +297,14 @@ def get_term_citations(accession, term_id):
 
 
 @bp.route("/<accession>/go/<term_id>/subfam")
+def get_signature_go_info(accession, term_id):
+    if accession.startswith("PTHR"):
+        return get_panther_go_subfam(accession, term_id)
+    elif accession.startswith("G3DSA"):
+        return get_funfam_go(accession, term_id)
+    else:
+        return jsonify(), 404
```

Comment:
Add an error with some details, and use the HTTP status code `400` (invalid request) instead of `404` (not found):

```suggestion
        return jsonify({
            "error": {
                "title": "Invalid signature",
                "message": "Only CATH-Gene3D and PANTHER signatures "
                           "have subfamilies."
            }
        }), 400
```